cip-constitution-tech/README.md

```diff
@@ -0,0 +1,255 @@
+There are six levels to headers with, each being defined via a extra `#`.
+Headers are ended via a line break.
+Headers SHOULD be followed below by a blank line.
+Headers SHOULD not be preceded by whitespace.
+
+The lower the number of `#` the larger order the text SHOULD be rendered.
+
+Example:
+```md
+# H1
+
+## H2
+
+### H3
+
+#### H4
+
+##### H5
+
+###### H6
+
+```
+
+If text is in a header no other formatting can be applied.
+
+#### Emphasis
+
+Emphasis is applied to text between single or double asterisks, without space between asterisks and text.
+Italicized emphasis is shown via single asterisk (`*`).
+Bold emphasis is shown via double asterisks (`**`).
+
+Emphasis cannot span multiple lines.
+
+Examples:
+```md
+Emphasis, aka italics, with single *asterisks*.
+
+Strong emphasis, aka bold, with double **asterisks**.
+```
+
+The text contained within headings can be emphasized.
+
+Both italicized and bold cannot be applied to the same text.
+
+#### Code and Syntax Highlighting
+
+Texted can be highlighted as code, when encased without spaces by backticks.
+This MUST not contain line breaks.
+
+Example:
+```md
+Inline `code` has `back-ticks around` it.
+```
+
+The text contained within headings or emphasis cannot be highlighted as code.
+
+
+#### Unordered Lists
+
+// todo
+
+#### Line Breaks / Paragraphs
+
+To create paragraphs, use a blank line to separate one or more lines of text.
+
+Examples:
+```md
+Here's a line for us to start with.
+
+This line is separated from the one above by two newlines, so it will be a *separate paragraph*.
+
+This line is also a separate paragraph, but...
+This line is only separated by a single newline, so it's a separate line in the *same paragraph*.
+```
+
+### Best Practices
+- when hash doesnt match
+- when unrecognised formatting/style
+- Keep lines left-aligned without unneeded whitespace or tabs
+- tabs vs spaces
+
+## Rationale: how does this CIP achieve its goals?
+
+Just like CIP-1694, we aim to define a minimal viable set of standards to ensure governance can be achieved.
+
+### Sentences
+- improve the readability of diff views
+
+### Versioning
+- why not build in options in hashalgo (like CIP100)
+- why a whole replacement?
+
+### File Type
+- just simple and easy
+
+### Hashing
+- we choose blake2b-256 as it is a standard for hashing in Cardano
+- standard, lots of tools already use it 
+
+### Storage
+- The constitution is the most important document for Cardano (?), ensuring its permissionless accessibly is paramount
+- Storing each iteration immutably is just as important, to allow access of past constitutions
+- IPFS is well known already with good tooling support -> improve accessibility
+
+### Rich Text Formatting
+- why not all markdown?
+- wanted to keep a bare minimum spec, no images or videos
+- also dont want people to be able to attack consumers by attacks utilizing the HTML
+- wanted to allow standard markdown libs to work
+
+## Open Questions
```

Comment:
Since this is only a general section for the CPS format I also assume these are author's notes and/or an editorial checklist so please @Ryun1 indicate if this information is to be kept in the final version of the document.

Reply:
Yeah, more of an editorial check list
I may or may not keep it as an optional section in the final version